src/ICSharpCode.SharpZipLib/Zip/ZipFile.cs

```diff
@@ -2553,11 +2549,11 @@ private int FindExistingUpdate(ZipEntry entry)
 			return result;
 		}
 
-		private int FindExistingUpdate(string fileName)
+		private int FindExistingUpdate(string fileName, bool isEntityName = false)
```

Comment:
Is this supposed to be `isEntryName`? Not sure why the word "entity" is used here?

Reply:
You are right, it should be `isEntryName`

Reply:
```suggestion
		private int FindExistingUpdate(string fileName, bool isEntryName = false)
		{
			int result = -1;

			string convertedName = GetTransformedFileName(fileName);
			string convertedName = !isEntryName ? GetTransformedFileName(fileName) : fileName;
```